README.md

```diff
@@ -271,15 +271,15 @@ console.log("This code block is linted normally.");
 
 The [`linter-eslint`](https://atom.io/packages/linter-eslint) package allows for linting within the [Atom IDE](https://atom.io/).
 
-In order to see `eslint-plugin-markdown` work its magic within Markdown code blocks in your Atom editor, you can go to `linter-eslint`'s settings and within "List of scopes to run ESLint on...", add the cursor scope "source.gfm".
+In order to see `@eslint/markdown` work its magic within Markdown code blocks in your Atom editor, you can go to `linter-eslint`'s settings and within "List of scopes to run ESLint on...", add the cursor scope "source.gfm".
 
-However, this reports a problem when viewing Markdown which does not have configuration, so you may wish to use the cursor scope "source.embedded.js", but note that `eslint-plugin-markdown` configuration comments and skip directives won't work in this context.
+However, this reports a problem when viewing Markdown which does not have configuration, so you may wish to use the cursor scope "source.embedded.js", but note that `@eslint/markdown` configuration comments and skip directives won't work in this context.
 
 ## Contributing
 
 ```sh
-$ git clone https://github.com/eslint/eslint-plugin-markdown.git
-$ cd eslint-plugin-markdown
+$ git clone https://github.com/eslint/markdown.git
+$ cd markdown
```

Comment:
We should change the same here:

https://github.com/eslint/eslint-plugin-markdown/blob/923e440094e334d2a0c89c8b9d58bc20ade86373/examples/react/README.md?plain=1#L14-L15

and here:

https://github.com/eslint/eslint-plugin-markdown/blob/923e440094e334d2a0c89c8b9d58bc20ade86373/examples/typescript/README.md?plain=1#L14-L15